build/docker/intel-vpu-plugin.Dockerfile

```diff
@@ -28,18 +28,18 @@ ARG BUILDFLAGS="-ldflags=-w -s"
 WORKDIR $DIR
 COPY . .
 
-RUN echo "deb-src http://deb.debian.org/debian buster main" | tee -a /etc/apt/sources.list
+RUN echo "deb-src http://deb.debian.org/debian sid main" | tee -a /etc/apt/sources.list
```

Comment:
Can we use 'unstable' instead of 'sid' here?

Reply:
changed.